mllib/src/main/scala/org/apache/spark/ml/classification/NaiveBayes.scala

```diff
@@ -227,6 +233,35 @@ class NaiveBayesModel private[ml] (
 
   @Since("1.6.0")
   override def write: MLWriter = new NaiveBayesModel.NaiveBayesModelWriter(this)
+
+  private var featureNames: Option[Array[String]] = None
```

Comment:
@mengxr I remove the previous `NaiveBayesSummary` and add these two `featureNames` and `labelNames` because we need these two variables to be accessed from `NaiveBayesModel`.